etc/neutron/services/f5/f5-openstack-agent.ini

```diff
@@ -409,35 +409,44 @@ f5_route_domain_strictness = False
 # if the same BIG-IP device is not being used
 # as the Neutron Router implementation.
 #
-# This setting will be forced to True if 
+# This setting will be forced to True if
 # f5_global_routed_mode = True.
 #
 f5_snat_mode = True
 #
-# This setting will specify the number of snat 
-# addresses to put in a snat pool for each 
+# This setting will specify the number of snat
+# addresses to put in a snat pool for each
 # subnet associated with a created local self IP.
-# 
-# Setting to 0 (zero) will set VIPs to AutoMap 
+#
+# Setting to 0 (zero) will set VIPs to AutoMap
 # SNAT and the device's local self IP will
 # be used to SNAT traffic.
 #
 # In scalen HA mode, this is the number of snat
 # addresses per active traffic-group at the time
 # a service is provisioned.
 #
-# This setting will be forced to 0 (zero) if 
+# This setting will be forced to 0 (zero) if
 # f5_global_routed_mode = True.
 #
 f5_snat_addresses_per_subnet = 1
+
+# This provider_name use to distinguish different
```

Comment:
Suppose the provider_name can have more usages in the future because it's an identifier of provider/plugin. 
So how about to say: 

```
# This provider_name is the identifier of f5driver. Currently it is used to distinguish ... 
```

Not sure of that, you decide.